lib/stack_master/template_compiler.rb

```diff
@@ -11,7 +11,7 @@ def self.compile(config, template_compiler, template_dir, template, compile_time
       compiler.require_dependencies
       compiler.compile(template_dir, template, compile_time_parameters, compiler_options)
     rescue StandardError => e
-      raise TemplateCompilationFailed.new("Failed to compile #{template} with error #{e}.\n#{e.backtrace}")
+      raise TemplateCompilationFailed, "Failed to compile #{template}"
```

Comment:
Remove the backtrace from the `TemplateCompilationFailed` error message. This was unreadable.

Fixes 260.